duckduckgo/macos-browser
# UI Polishing

Task/Issue URL: https://app.asana.com/0/0/1200636627320656/f

**Description**:
UI of navigation bar, address bar and suggestions polished 


**Steps to test this PR**:
1. Please see [Figma design](https://www.figma.com/file/vfBLB1eDKGlJqZCKxPyneL/Address-Bar-Revision-June-2021?node-id=2%3A2350) and make sure UI follows the new spec
(I synced with Bryan yesterday and he was OK with the result. We also changed few more icons)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
